FAQ.md

```diff
@@ -0,0 +1,94 @@
+## FAQ
```

Comment:
Could you use lowercase for the file names please? Thanks for contributing!

Reply:
It's fixed!